content/06-vets-developer-docs/04-vets-website/03-forms/07-form-map.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Form Map - Where Does This Stuff Come From?
+label: Form Map
+---
+In this doc, we'll explain where the content you see on the form actually comes from. The file locations in this doc assumes you created a simple form using the Yeoman generator, saving the app in `new-form` and giving it a root URL of `/new-form` (questions 2 and 4 in the generator).
+
+## Form Introduction Page
+
+![](/va-digital-services-platform-docs/assets/develop/images/forms/form-map-1.png)
```

Comment:
Should we use something like:
```
<img src="{{ assetPath }}img/docs/state-info.png" width="208">
```
As in `content/03-define-and-design/_10-messaging-and-errors/01-messaging-design-guide.md`?

Reply:
I'm not sure. I copied the way the image links were set up on form-tutorial page and those image [definitely show up on the live site](https://department-of-veterans-affairs.github.io/va-digital-services-platform-docs/docs/vets-developer-docs/vets-website/forms/form-tutorial.html). Since this new page is a sibling of the tutorial page, the image links I included should also work.

Reply:
Thanks for clarifying, sounds good.

Reply:
Should we add some alt text here?

Reply:
sure, I'll do that.